fuzzers/forkserver/fuzzbench_forkserver_cmplog/justfile

```diff
@@ -0,0 +1,111 @@
+# Justfile for LibAFL cmplog demo
+# Usage: just <command>
+
+# Default recipe - show available commands
+default:
+    @just --list
+
+# Compile the test targets
+compile:
+    @if ! command -v afl-clang-fast >/dev/null 2>&1; then \
```

Comment:
then you have to install it in this script

Reply:
Should we really install dependencies in justfiles? Maybe document them?

Reply:
maybe no.
but then the installation should be done before this file is run in github ci